Editor/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,92 @@
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+using Cinemachine.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Provides a simple API to create a button that can add a CinemachineInputProvider component with a default
+    /// input asset controlling XY axis from the InputSystem package to the gameObject if it does not
+    /// already have CinemachineInputProvider.
+    /// </summary>
+    class CinemachineDefaultMouseInput
+    {
+        static CinemachineDefaultMouseInput s_Instance;
+        InputActionReference m_InputActionReference = null;
+        
+        /// <summary>
+        /// Initialize-on-demand singleton.
+        /// </summary>
+        /// <returns>Initialized instance</returns>
+        public static CinemachineDefaultMouseInput GetInstance() {
+            if (s_Instance == null) {
+                s_Instance = new CinemachineDefaultMouseInput();
+            }
+            return s_Instance;
+        }
+
+        CinemachineDefaultMouseInput()
+        {
+            var inputActionAsset = AssetDatabase.LoadAssetAtPath<InputActionAsset>("Packages/com.unity.inputsystem/" +
+                "InputSystem/Plugins/PlayerInput/DefaultInputActions.inputactions");
+
+            using var inputActions = inputActionAsset.GetEnumerator();
+            var hasNext = true;
+            while (hasNext)
+            {
+                if (inputActions.Current != null &&
+                    inputActions.Current.ToString() == "Player/Look[/Mouse/delta,/Pen/delta]")
```

Comment:
It's not finding this action for me, the string is not the same.  Debugging into it, mine says "Player/Look[/Mouse/delta,/Touchscreen/delta]".
Maybe just compare the prefix and match "Player/Look" ?
What is the right approach?  Please validate with the input people.

Reply:
I'll check with them.